### PR TITLE
Prebid.js Video

### PR DESCRIPTION
**Pull request by ix-prebid-support**
## Type of change

- [x] Feature

## Description of change

Configured the IX adapter for the VIDEO ad type with instream and outstream context support.

- contact email of the adapter’s maintainer
pdu-supply-prebid@indexexchange.com

- [x] official adapter submission

### CODE REVIEW

**Comment by bretg** [comment]
@ix-prebid-support - please address build failure.

**Comment by ix-prebid-support** [comment]
Hi @bretg thank-you for flagging we are working to resolve this test case issue and will report back.

**Comment by webdeveloperpr** [comment]
Waiting for this to be added.

File: modules/ixBidAdapter.js [hunk 1]
```diff
@@ -133,6 +181,145 @@ function isValidBidFloorParams(bidFloor, bidFloorCur) {
...
+ *
+ * @param  {*}      id          Id of the impression.
+ * @param  {array}  impressions List of impressions sent in the request.
+ * @return {object}             The impression with the associated id.
+ */
+function getBidRequest(id, impressions) {
+  if (!id) {
+    return;
+  }
+  return impressions.find(imp => {
```
[the remark this answers is not in the document]
Comment (ix-prebid-support) [reply modules/ixBidAdapter.js]: Okay this was very helpful, thank-you kindly! Let's see how the next round of checks go :)

**Comment by ix-prebid-support** [comment]
@bretg thank-you for your patience the issues have been resolved but we still have an outstanding review from @jaiminpanchal27 otherwise things look good to go here.

**Pull request merged**